lib/LedgerSMB/Report/GL.pm

```diff
@@ -333,7 +333,7 @@ sub run_report{
     my @rows = $self->call_dbmethod(funcname => 'report__gl');
     for my $ref(@rows){
         if ($ref->{amount} < 0){
-            $ref->{debits} = $ref->{amount} * -1;
+            $ref->{debits} =  -$ref->{amount};
```

Comment:
`-$ref->{amount}`  inverts the sign of `$ref->{amount}`. 

I think the direct unary minus alone is clearer.

My fonts are bigger than most peoples.   `0 - $stuff` also bears consideration.